src/gui/AutomationPatternView.cpp

```diff
@@ -270,17 +269,22 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 	if( !( m_pat->getTrack()->isMuted() || m_pat->isMuted() ) )
 		c = styleColor;
 	else
-		c = QColor( 80, 80, 80 );
+		c = mutedColor();
 
 	if( isSelected() == true )
 	{
-		c.setRgb( qMax( c.red() - 128, 0 ), qMax( c.green() - 128, 0 ), 255 );
+		c = selectedColor();
 	}
 
 	lingrad.setColorAt( 1, c.darker( 300 ) );
 	lingrad.setColorAt( 0, c );
-
-	p.setBrush( lingrad );
+	
+	if ( gradient() ) {
+		p.setBrush( lingrad );
+	} else {
+		p.setBrush( c );
+	}
+	
```

Comment:
>p.setBrush(gradient() ? lingrad : c);

Already tried that, and it is not allowed for the operands to be of a different type (QColor vs QLinearGradient)

I'll look into fixing the other stuff